lib/workos/organizations.rb

```diff
@@ -180,6 +180,32 @@ def delete_organization(id:)
         response.is_a? Net::HTTPSuccess
       end
 
+      # Retrieve a list of roles for the given organization.
+      #
+      # @param [String] organizationId The ID of the organization to fetch roles for.
+      def list_organization_roles(organization_id:)
+        response = execute_request(
+          request: get_request(
+            path: "/organizations/#{organization_id}/roles",
+            auth: true,
+          ),
+        )
+
+        parsed_response = JSON.parse(response.body)
+
+        roles = parsed_response['data'].map do |role|
+          ::WorkOS::Role.new(role.to_json)
```

Comment:
Is the leading `::` needed here?

```suggestion
          WorkOS::Role.new(role.to_json)
```

Reply:
Oops, nope. Fixing, thanks!